apps/forms-flow-ai/forms-flow-web/package.json

```diff
@@ -41,6 +44,8 @@
     "jsonwebtoken": "^8.5.1",
     "keycloak-js": "^10.0.2",
     "lodash": "^4.17.21",
+    "moment": "^2.29.1",
```

Comment:
Can we use a different dependency than moment?

Moment is deprecatd.  They do not suggest using it anymore, and only to use it in legacy projects.  Even Chrome itself will start showing warnings in dev toos to remove Moment.  https://momentjs.com/docs/#/-project-status/

It is one of the largest dependencies on projects, and a lot of the date logic can be used in much smaller libraries (like `date-fns` - https://date-fns.org/).

Reply:
I will take this as a technical debt and will work on this asap.